cmd/login/login.go

```diff
@@ -71,20 +84,27 @@ func NewLoginCmd() *cobra.Command {
 	return cmd
 }
 
-func login(authProvider, username, password, providerID string) (string, error) {
-	JSONClient, err := jsonclient.New(jsonclient.Options{
-		BaseURL: authProvider,
-	})
+func login(cfg loginConfig) (string, error) {
+	clientOptions := jsonclient.Options{
```

Comment:
The login command could be put inside the sdk, in order to avoid the code duplication?

Reply:
Code refactoring has been performed. In theory code duplication should be reduced.